queries.sql

```diff
@@ -12,4 +12,18 @@ select name from animals where  extract(year from date_of_birth) between 2016 an
   select avg(weight_kg) from animals;
   select count(neutered) from animals group by neutered;
   select Max(weight_kg), Min(weight_kg) from animals group by species;
-   select avg(escape_attemps) from animals where extract(year from date_of_birth) between 1990 and 2000 group by species;
\ No newline at end of file
+   select avg(escape_attemps) from animals where extract(year from date_of_birth) between 1990 and 2000 group by species;
+  /*What animals belong to Melody Pond?*/
+   select a.name from animals a join owners o on o.id = a.owner_id where o.full_name = 'Melody Pond';
+   /*List of all animals that are pokemon (their type is Pokemon).*/
+   select * from animals a join species s on s.id = a.species_id where s.name = 'Pokemon';
+   /*List all owners and their animals, remember to include those that don't own any animal.*/
+    select o.full_name, a.name from animals a join owners o on o.id = a.owner_id where o.id = owner_id;
+    /*How many animals are there per species?*/
+   select count(a.name)from animals a join species s on s.id = species_id group by s.id;
+   /*List all Digimon owned by Jennifer Orwell.*/
+   select a.name, s.name, o.full_name from animals a join species s on s.id =a.species_id join owners o on o.id = a.owner_id where o.full_name = 'Jennifer Orwell' and s.name = 'Digimon';
+   /*List all animals owned by Dean Winchester that haven't tried to escape.*/
+   select a.name, a.escape_attemps,o.full_name from animals a join owners o on o.id = a.owner_id where a.escape_attemps = 0 and o.full_name = 'Dean Winchester';
+   /*Who owns the most animals?*/
+   select count(a.name),o.full_name from animals a join owners o on o.id = a.owner_id group by o.full_name;
```

Comment:
- [ ] **Optional:** Great job. To be more explicit, you could order the query by DESC order to ensure they are listed in the specified order.